### PR TITLE
Fixed wrong re-direction of portfolio link on Resume page

### PR DESCRIPTION
**Pull request by RashidSiddiqui2004**
Fixes Issue #96 

Blog Link/Portfolio Link Not Functioning Correctly

Now, **_it re-directs to correct page:_**

resolve

### CODE REVIEW

**Comment by dhairyathedev** [comment]
@RashidSiddiqui2004 Replace the link component with following: 
```jsx
                <Link
                  href={
                    profile.blog.includes("http")
                      ? profile.blog
                      : `https://${profile.blog}`
                  }
                  target="_blank"
                  rel="noopener noreferrer"
                  className="text-blue-600 hover:underline"
                >
                  {profile.blog}
                </Link>
```

**Comment by RashidSiddiqui2004** [comment]
@dhairyathedev Ok, I was working on the same, thank you.

**Comment by ashutosh-rath02** [comment]
@dhairyathedev review?

**Review by dhairyathedev — CHANGES_REQUESTED** [review]
Don't include `{profile.blog.split("/").pop()}` and revert back to original `{profile.blog}`

File: components/Resume.tsx [hunk 1]
```diff
@@ -199,7 +199,7 @@ const Resume = () => {
               )}
               {showBlog && (
                 <Link
-                  href={`https://${profile.blog}`}
+                  href={profile.blog}
```
[the remark this answers is not in the document]
Comment (RashidSiddiqui2004) [reply components/Resume.tsx]: @dhairyathedev Ok I'll commit with the above code.

**Review by dhairyathedev — APPROVED** [review]

**Pull request merged**